.gitignore

```diff
@@ -1,9 +1,171 @@
-.idea/
-.vscode/
-*.iml
-.env
-.DS_Store
-venv/
+# Created by https://www.toptal.com/developers/gitignore/api/python
```

Comment:
Attach screenshots of all created or modified pages, I believe there is more than 2